scripts/scil_surface_create.py

```diff
@@ -0,0 +1,170 @@
+    mxg.add_argument('--in_mask',
+                     help='Path of the mask (nii or nii.gz).')
+
+    p.add_argument('out_surface',
+                   help='Output surface (.vtk)')
+
+    p.add_argument('--indices', nargs='+',
+                   help='List of labels indices to use for the surface.')
+
+    p.add_argument('--value', default=0.5,
+                   type=ranged_type(float, 0, None, min_excluded=False),
+                   help='Isosurface threshold value used. '
+                        'This value is called isovalue '
+                        'in mbcube. [%(default)s]')
+
+    morpho_g = p.add_argument_group('Morphology options')
+    morpho_g.add_argument('--smooth',
+                          type=ranged_type(float, 0, None, min_excluded=False),
+                          help='Smoothing size with'
+                               ' 1 implicit step. [%(default)s]')
+    morpho_g.add_argument('--erosion',
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Erosion: number of iterations. [%(default)s]')
+    morpho_g.add_argument('--dilation',
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Dilation: number of iterations. [%(default)s]')
+    morpho_g.add_argument('--opening',
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Opening (dilation of the erosion): number '
+                               'of iterations. [%(default)s]')
+    morpho_g.add_argument('--closing',
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Closing (erosion of the dilation): number '
+                               'of iterations. [%(default)s]')
+
+    p.add_argument('--fill', action='store_true',
+                   help='Fill holes in the image.')
+
+    p.add_argument('--vox2vtk', action='store_true',
+                   help='Transformation to vox2vtk.')
+
+    add_verbose_arg(p)
+    add_overwrite_arg(p)
+
+    return p
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+    logging.getLogger().setLevel(logging.getLevelName(args.verbose))
+
+    assert_inputs_exist(parser, [], [args.in_labels, args.in_mask])
+    assert_outputs_exist(parser, args, args.out_surface)
+
+    if args.in_labels:
+        # Load volume
+        labels_img = nib.load(args.in_labels)
+        labels_volume = get_data_as_labels(labels_img)
+
+        # Removed indices
+        if args.indices:
+            mask = merge_labels_into_mask(labels_volume,
+                                          ' '.join(args.indices))
+        else:
+            logging.warning('No indices provided, '
+                            'it will use all indices.')
+            mask = merge_labels_into_mask(labels_volume,
+                                          "1:" + str(labels_volume.max()))
```

Comment:
Personally, I think this is not the best, fastest nor prettiest to get a mask from all indices.
"mask = labels_volume > 0" or "mask = labels_volume >= 1" should do the job.